common/lib/xmodule/xmodule/video_module/video_handlers.py

```diff
@@ -276,7 +276,7 @@ def transcript(self, request, dispatch):
 
         elif dispatch.startswith('available_translations'):
 
-            available_translations = self.available_translations(transcripts)
+            available_translations = self.available_translations(transcripts, verify_assets=True)
```

Comment:
So what's the idea of falling back to english if we still forcing some modules to verify/not-verify the assets?